stdlib/@python2/typing.pyi

```diff
@@ -77,7 +77,7 @@ Counter = _Alias()
 Deque = _Alias()
 
 # Predefined type variables.
-AnyStr = TypeVar("AnyStr", str, unicode)
+AnyStr = TypeVar("AnyStr", str, unicode)  # noqa Y001
```

Comment:
```suggestion
AnyStr = TypeVar("AnyStr", str, unicode)  # noqa: Y001
```

Reply:
Heh, I actually just copied and pasted the `# noqa` comment from `_typeshed.Self`

Reply:
Maybe it works with both.